### PR TITLE
Completed Wine. Rb

### PR DESCRIPTION
**Pull request by tstwine**
Comfort level 3.5  More HW submissions to come.

### CODE REVIEW

File: class03/tina/wine.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ 	multi_word_wines = []
+ 	multi_word_wines = wine_cellar.select do |wine|
+ 		wine [:label] = "multi_word_wines" 
+ 	end		
+
+  labels_pinot_noir = multi_word_wines.select do |label|
+        labels[:label]
+  end      
+
+
```
Comment (mobjohnson) [inline class03/tina/wine.rb]: Looks Good!

**Comment by mobjohnson** [comment]
I cannot accept this pull request as it includes a file outside of your student folder. I have reviewed your homework and show you as completed for homeworks 1 and 2.

**Comment by tstwine** [comment]
Martin,
I'm confused. What do you mean outside my student folder?

**Comment by mobjohnson** [comment]
You have a student folder in each of the classes that is your first name.
Your pull request included files that were edited elsewhere.

In the future, before doing $git add .

make sure you are in your student folder under the class## folder. This
will keep git from adding files in the directory structure above the
student folder.

*Martin Johnson*
Full-Stack Web Developer
mobjohnson.com

**Comment by tstwine** [comment]
I'm not clear on this. I thought I was working out of my student folder
I thought I did that because I followed the instructions on GitHub. I'm not
quite sure how or what happened. What's going to happen regarding the
wine.rb HW assignment?  I'll resubmit in my folder so I can get credit for
it.

**Comment by mobjohnson** [comment]
I have reviewed your homework and show you as completed for wines.rb. You
do not need to resubmit.

*Martin Johnson*
Full-Stack Web Developer
mobjohnson.com

**Comment by tstwine** [comment]
Ok.

**Pull request closed without merging**